automobile/data/product_attribute.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_attribute_1" model="product.attribute">
+        <field name="name">Brand</field>
+        <field name="product_tmpl_ids" eval="[(6, 0, [ref('automobile.product_template_12')])]"/>
```

Comment:
```suggestion
```

product.attribute not needed product_tmpl_ids as it is auto computed based on product.template.attribute.line